apps/frontend/src/libs/hooks/use-pagination/libs/helpers/get-initial-page.helper.ts

```diff
@@ -0,0 +1,20 @@
+import { DEFAULT_VALUES, FIRST_PAGE } from "../constants/constants.js";
+import { isNumberInRange, parseQueryParameterToNumber } from "./helpers.js";
+
+const getInitialPage = (
+	pageQueryParameter: null | string,
+	totalPages: number,
+): number => {
+	const pageParameter = parseQueryParameterToNumber(pageQueryParameter);
+
+	if (
+		pageParameter !== null &&
+		isNumberInRange(pageParameter, { max: totalPages, min: FIRST_PAGE })
+	) {
```

Comment:
I think you don't need parseQueryParameterToNumber, isNumberInRange, you can simplify code without them
```suggestion
	const page = Number(pageQueryParameter);

	if (
		Number.isInteger(page) &&
		page <= totalPages &&
		page >= FIRST_PAGE 
	) {
```

Reply:
Yeah, thats definitely better